digdag-standards/src/main/java/io/digdag/standards/operator/pg/PgConnection.java

```diff
@@ -116,7 +119,7 @@ protected StatusRow lockStatusRow(UUID queryId)
         {
             try (Statement stmt = connection.createStatement()) {
                 ResultSet rs = stmt.executeQuery(String.format(ENGLISH,
-                            "SELECT completed_at FROM %s WHERE query_id = '%s' FOR UPDATE",
+                            "SELECT completed_at FROM %s WHERE query_id = '%s' FOR UPDATE NOWAIT",
```

Comment:
I think you need to check SQLError because `NOWAIT` throws exception if the row is locked, according to postgresql's document.